frontend/layouts/static-page/header/component.tsx

```diff
@@ -1,5 +1,6 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback, useEffect } from 'react';
```

Comment:
🟠

The changes you did here broke the story. To fix it, wrap the `{Story()}` line with `<QueryClientProvider client={queryClient} />` in `.storybook/preview.js`. You can see how it's done in the application in `pages/_app.tsx`.